Confus/Confus/Health.h

```diff
@@ -10,6 +10,7 @@ namespace Confus
 		std::function<bool(irr::scene::ISceneNode* a_DamagedNode)> m_DeathCallback;
 		irr::scene::ISceneNode* m_AttachedNode;
 		int m_Health;
+		int m_MaxHealth = 100;
```

Comment:
Should Health & MaxHealth really be a private member variable? I think we might want to access this in the future (power ups or health checks?). Would like a second opinion on this.